src/components/alert/alert.svelte

```diff
@@ -21,20 +24,18 @@
   export let isToast = false
   export let hasActions = $$slots.actions
   export let hasContentAfter = $$slots['content-after']
+  export let hasTitle = $$slots.title
```

Comment:
Hey @petemill what's the point of exporting this? I feel like it also might be slightly confusing to set `mode=simple` and provide a title and have the alert show in `full` mode?

I think at least we shouldn't export this :smile:

Additionally, I think `showTitle` is probably a better name, as we hide the title when this is false (and we might have a title, and just decide not to show it)

Reply:
Well yes I was confused as to why we are exporting `hasActions` and `hasContentAfter` and guessed it was for svelte (and storybook), in order to hide the title element when no title is provided.